app/screens/PrivacyLock.tsx

```diff
@@ -61,6 +61,9 @@ export function PrivacyLock (): JSX.Element | null {
 function authenticateOrExit (localAuth: LocalAuthContext): void {
   const backHandler = BackHandler.addEventListener('hardwareBackPress', () => null)
   localAuth.privacyLock()
-    .catch(e => BackHandler.exitApp())
+    .catch(async () => {
+      await AppLastActiveTimestamp.forceRequireReauthenticate()
```

Comment:
The key change is here, when user failed to authenticate. Force the timeout back to older timestamp, thus user cannot skip authentication if resume app within 1minute.